l5kit/l5kit/rasterization/render_context.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+
+class RenderContext:
+    def __init__(
+        self, raster_size_px: np.ndarray, pixel_size_m: np.ndarray, center_in_raster_ratio: np.ndarray,
+    ) -> None:
+        """
+        This class stores render context information (raster size, pixel size, raster center / principle point) and
```

Comment:
Is 'render' actually just another word for 'rasterizer'? Does it make sense to introduce this, adding a bit of 'confusion' by multiple terms for the same?

Reply:
We use this notation internally, so we're trying to be consistent here :)